src/shared/contentBox/ContentBox.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import styled from "styled-components";
+
+const ContentBoxStyled = styled.div`
+  width: 940px;
+  padding: 20px 100px 20px 100px;
```

Comment:
Ik ben niet erg bekend met styled-components, maar is het misschien een idee om net als bij CSS code alles te baseren op variabelen die je ergens 1x defined? Dus:

`padding: spacing-normal 5*spacing-normal spacing-normal 5*spacing-normal `